server/views/components/top-bar/template.njk

```diff
@@ -2,11 +2,38 @@
 
 <header class="govuk-header top-bar-container" data-module="header">
   <div class="govuk-width-container top-bar">
-    <img class="hub-logo" src="/public/images/icons/content-hub.png" alt="logo">
-    <a href="/" class="govuk-heading-m govuk-link govuk-link--inverse">Content Hub <span>{{ params.establishmentDisplayName }}</span></a>
-    {% if params.showSearch %}
-      {{ hubSearch({ query: params.searchQuery, small: true }) }}
-    {% endif %}
+    <div class="top-bar__title">
+      <img class="hub-logo" src="/public/images/icons/content-hub.png" alt="logo">
+      <a href="/" class="govuk-heading-m govuk-link govuk-link--inverse">{{ t('topBar.title') }} <span>{{ params.establishmentDisplayName }}</span></a>
+    </div>
+    <nav aria-label="Content Hub Navigation" class="top-bar__nav">
+      <div class="top-bar__navigation">
+        <h2 id="top-bar__navigation-heading" class="govuk-visually-hidden">
+          ::before
+          " Content Hub Navigation Menu "
```

Comment:
Presuming this needs a translation?

Reply:
Yes - this is out of scope for this ticket though